templates/studygroups/email/community_digest.html

```diff
@@ -261,6 +262,7 @@ <h2>
           <td>{{ studygroup.facilitator.first_name | title }}</td>
           <td><a href="{% url 'studygroups_course_page' studygroup.course.pk %}">{{ studygroup.course.title }}</a></td>
           <td>{{ studygroup.city | title }}</td>
+          <td>{{ studygroup.team }}</td>
```

Comment:
`studygroup.team` will call the `__str__` function to get a string, in this case, just the name of the team. In general, `__str__` isn't intended to be public facing, so maybe we should explicitly use `studygroup.team.name`.